documentation/docs/user_guide/inputs/cable_nml.md

```diff
@@ -84,6 +84,76 @@ applications. The following are annotated examples of cable.nml:
 &end
 ```
 
+| Namelist variable            | Type | Available values                               | Default value                                                          | Description                                                                                              |
+|------------------------------|------|------------------------------------------|------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| filename%met                 | character |                                          | '/projects/access/CABLE-AUX/offline/TumbaFluxnet.1.3_met.nc'                 | Meteorological forcing file                                                                                                 |
+| filename%out                 | character |                                          | 'out_cable.nc'                                                               | Output file                                                                              |
+| filename%log                 | character |                                          | 'log_cable.txt'                                                              | Log file                                                                                              |
+| filename%restart_in          | character |                                          | ' '                                                                          | Input restart file                                                                                                 |
+| filename%restart_out         | character |                                          | './restart_out.nc'                                                           | Output restart file                                                                                       |
+| filename%type                | character |                                          | '/projects/access/CABLE-AUX/offline/gridinfo_CSIRO_1x1.nc'                   | Gridinfo file                                                                                      |
+| filename%veg                 | character |                                          | '/projects/access/CABLE-AUX/core/biogeophys/veg_params_vbeta1.txt'           | Vegetation parameters file (obsolete, vegetation parameters are now in the pft_params.nml file)     |
+| filename%soil                | character |                                          | '/projects/access/CABLE-AUX/core/biogeophys/def_soil_params.txt'             | Soil parameters file ((obsolete, vegetation parameters are n:wow in the cable_soilparm.nml file)    |
```

Comment:
These two options are deprecated and have no effect anymore. They are still everywhere in the namelist files that are publicly available. Should we add a column for "deprecated"?

Reply:
Now that Ramzi has created a deprecated features section, wouldn't this be a good candidate to put in there?

Reply:
Sure, but then we need to:

- add a note at the top of the page to check the deprecated feature page if people have a file with options that are not listed here. [In an admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage).
- remove these entries from the annotated examples as well.